shared/common-adapters/icon.constants.js

```diff
@@ -0,0 +1,265 @@
+// @flow
+// This file is GENERATED by npm run updated-fonts. DON'T hand edit
+
+export const FontIcons = {
```

Comment:
Do we use it in code currently?

I guess we can always change back to making an enum, just a bit wary of using `$Enum` and `typeof` on an object.